docs/_docs/About us/How-we-make-decisions.md

```diff
@@ -0,0 +1,164 @@
+# Context
+
+During batch 4, we identified several challenges with how we currently make decisions throughout the organizations. Gathering insights and feedback from multiple people from multiple places, the followings are the main ones we are facing right now:
+
+### 1. Small decisions take a long time to close
+Up to now, we’ve been leaning towards a consensus approach to decision making.  While this is great to hear from everyone’s opinions, not every single decision is worth a discussion. This leads to proposals for improvements taking a long time to close and therefore be fixed.
+This creates frustration for the team, and especially the person who creates the proposals. In some cases, the person even had to chase up the decision makers for final approvals. 
+
+### 2. While some important decisions were closed without us hearing from everyone
+On the contrary, we also encountered cases where the exec team discussed and closed important decisions without first hearing opinions from the teams.
+While we are always happy to discuss when everyone raises objections, this makes people feel unheard and not invited to participate.
+
+### 3. Team members don’t know to whom to bring up the problems
+We experienced inter-AOR issues that would require collaboration and sign-off from multiple AORs. Given our lack of documentation on the decision making process, it’s hard to know who to bring up the problems and who would have a final call on it.
+
+# Decision makers
+Unlike hierarchical organizations where most decisions are closed by managers or executives, at LDSA, we believe that decisions should be made by the one with the most expertise and with most visibility about the issues. Depending on the type of decisions, that can be the AOR Lead, or the Exec team.
```

Comment:
Added that AOR members can also close decisions within the AOR, and remove that sentence. Agree with you that you should not wait for AOR Lead to come and "close" decision. At the same time, we'd still need some types of structure, as otherwise, we will experience "different people saying different things". As long as the responsibilities are clear, it should be okay